### ISSUE TITLE
Panic when opening Netrw

### ISSUE THREAD

**Issue by badosu** [vhakulinen/gnvim#24]
### How to reproduce

- `mv .config/nvim/init.vim .config/nvim/init.vim_` (optional, bring the config back afterward)
- Open gnvim with the terminal
- Type `:e .`

### What happens

The program panics leaving the output:

```
thread 'main' panicked at 'byte index 2 is out of bounds of ``', src/libcore/str/mod.rs:2085:9
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

### What is expected

Vim's netrw is opened.

**Comment by badosu** [comment]
Could not reproduce after a restart

**Comment by alexlai97** [comment]
> Could not reproduce after a restart

After I mv my init.vim, open gnvim, :edit . , it produces
`thread 'main' panicked at 'byte index 2 is out of bounds of ``', src/libcore/str/mod.rs:2010:9`

I found that when there are few files in the directory, it won't panic, however, when I try to explore my home directory (where there are a lot of dot files, directories + dotfiles = 137) using netrw, it panics and produces error above.

**Comment by vhakulinen** [comment]
@alexlai97 #30 did fix the issue with me. Can you test if it fixes it for you too?

**Comment by alexlai97** [comment]
> @alexlai97 #30 did fix the issue with me. Can you test if it fixes it for you too?

Both fixed #24, #29.
